edge-proxy/deb/debian/edge-proxy.service

```diff
@@ -1,7 +1,5 @@
 [Unit]
 Description=Edge Proxy
-Requires=wait-for-pelion-identity.service
-After=wait-for-pelion-identity.service
 
```

Comment:
I think that's supposed to be a separate PR?
@costanic ??

Reply:
That's unclear to me since it depends on how edge-core is configured at runtime.  I think if edge-proxy isn't up when edge-core tries to connect (if so configured), then the attempt will fail but edge-core should retry.  On the other hand, edge-proxy should be able to start without edge-core after this new feature is complete, so it should be fine.  No strong opinion either way at this point.

Reply:
Makes sense.... let's leave out my suggested edge-core.service changes.